### PR TITLE
V2subscription fixes

### PR DESCRIPTION
**Pull request by piotrmilcarz**

### CODE REVIEW

File: tests/integration/components/billing/warning-message-test.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module('Integration | Component | warning-message', function (hooks) {
       name: 'Startup',
       startingPrice: 3000,
       startingUsers: 10,
-      addon_configs: [
+      addonCsonfigs: [
```
Comment (artursmirnov) [inline tests/integration/components/billing/warning-message-test.js:58]: Looks like a typo to me. Why didn't it break anything?

**Review by artursmirnov — CHANGES_REQUESTED** [review]

File: tests/integration/components/billing/warning-message-test.js (hunk 1 above)
Comment (piotrmilcarz) [reply tests/integration/components/billing/warning-message-test.js]: Yep, it's a typo. In this `other addons` test probably not used.

Comment (artursmirnov) [reply tests/integration/components/billing/warning-message-test.js]: If it's not used, would it make sense to remove it then?

Comment (piotrmilcarz) [reply tests/integration/components/billing/warning-message-test.js]: Indeed, there is a sense.

**Review by artursmirnov — APPROVED** [review]
Looks good to me, thank you @piotrmilcarz

**Pull request merged**